packages/react-dom/src/__tests__/ReactDOMServerIntegrationAttributes-test.js

```diff
@@ -497,9 +497,9 @@ describe('ReactDOMServerIntegration', () => {
       itRenders(
         'badly cased aliased HTML attribute with a warning',
         async render => {
-          const e = await render(<meta httpequiv="refresh" />, 1);
-          expect(e.hasAttribute('http-equiv')).toBe(false);
-          expect(e.getAttribute('httpequiv')).toBe('refresh');
+          const e = await render(<form acceptcharset="utf-8" />, 1);
```

Comment:
Interesting change. Why?

Reply:
The meta doesn’t match any valid resource type (no key) so it gets omitted. This may be wrong but it’s helpful on the client if we can assume metas are resources. 

That said I could have just added a keyable property… I just replaced it with a functionally equivalent test